### PR TITLE
mail-client/clawsker: bump to version 1.3.5

### PR DESCRIPTION
**Pull request by kuraga**
mail-client/clawsker-1.3.5 is out: https://git.claws-mail.org/?p=clawsker.git;a=log;h=refs/tags/1.3.5

UPD: Also: updated to EAPI-8, migrated `xdg-utils` -> `xdg`.

### CODE REVIEW

File: mail-client/clawsker/clawsker-1.3.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+inherit xdg-utils
+
+DESCRIPTION="Applet to edit Claws Mail's hidden preferences"
+HOMEPAGE="https://www.claws-mail.org/clawsker.php"
+SRC_URI="https://www.claws-mail.org/tools/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (thesamesam) [inline mail-client/clawsker/clawsker-1.3.5.ebuild:14]: [ekeyword ~all clawsker-1.3.5.ebuild](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades)

File: mail-client/clawsker/clawsker-1.3.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline mail-client/clawsker/clawsker-1.3.5.ebuild:4]: Try 8.

File: mail-client/clawsker/clawsker-1.3.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	>=dev-perl/File-Which-1.210
+	mail-client/claws-mail
+"
+DEPEND="test? ( dev-perl/Test-Exception )"
+
+src_install() {
+	emake install DESTDIR="${D}" PREFIX=/usr
+}
+
+pkg_postinst() {
```
Comment (thesamesam) [inline mail-client/clawsker/clawsker-1.3.5.ebuild:31]: Drop both of these functions and change the inherit to `inherit xdg` which exports these phases for us.

File: mail-client/clawsker/clawsker-1.3.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+RDEPEND="
+	dev-lang/perl
+	dev-perl/Gtk3
+	dev-perl/Locale-gettext
+	>=dev-perl/File-Which-1.210
+	mail-client/claws-mail
+"
+DEPEND="test? ( dev-perl/Test-Exception )"
+
+src_install() {
```
Comment (thesamesam) [inline mail-client/clawsker/clawsker-1.3.5.ebuild:27]: I wonder, if you drop this, does it install?

File: mail-client/clawsker/clawsker-1.3.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-lang/perl
+	dev-perl/Gtk3
+	dev-perl/Locale-gettext
+	>=dev-perl/File-Which-1.210
+	mail-client/claws-mail
+"
+DEPEND="test? ( dev-perl/Test-Exception )"
```
Comment (thesamesam) [inline mail-client/clawsker/clawsker-1.3.5.ebuild:25]:
```suggestion
BDEPEND="test? ( dev-perl/Test-Exception )"
```

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: mail-client/clawsker/clawsker-1.3.5.ebuild (hunk 1 above)
Comment (kuraga) [reply mail-client/clawsker/clawsker-1.3.5.ebuild:14]: Yes, that was silly. Lost my abillities...

**Comment by kuraga** [comment]
@thesamesam , updated.

**Review by juippis — COMMENTED** [review]
You'll need to revert some parts of your change:
```
 * QA Notice: The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/local
 * 
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
```
```
# equery f mail-client/clawsker
 * Searching for clawsker in mail-client ...
 * Contents of mail-client/clawsker-1.3.5:
/usr
/usr/local
/usr/local/bin
/usr/local/bin/clawsker
/usr/local/share
/usr/local/share/applications
/usr/local/share/applications/clawsker.desktop
/usr/local/share/icons
/usr/local/share/icons/hicolor
/usr/local/share/icons/hicolor/128x128
/usr/local/share/icons/hicolor/128x128/apps
/usr/local/share/icons/hicolor/128x128/apps/clawsker.png
/usr/local/share/icons/hicolor/48x48
/usr/local/share/icons/hicolor/48x48/apps
/usr/local/share/icons/hicolor/48x48/apps/clawsker.png
/usr/local/share/icons/hicolor/64x64
/usr/local/share/icons/hicolor/64x64/apps
/usr/local/share/icons/hicolor/64x64/apps/clawsker.png
/usr/local/share/locale
/usr/local/share/locale/ca
/usr/local/share/locale/ca/LC_MESSAGES
/usr/local/share/locale/ca/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/da
/usr/local/share/locale/da/LC_MESSAGES
/usr/local/share/locale/da/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/de
/usr/local/share/locale/de/LC_MESSAGES
/usr/local/share/locale/de/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/es
/usr/local/share/locale/es/LC_MESSAGES
/usr/local/share/locale/es/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/fr
/usr/local/share/locale/fr/LC_MESSAGES
/usr/local/share/locale/fr/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/nb
/usr/local/share/locale/nb/LC_MESSAGES
/usr/local/share/locale/nb/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/nl
/usr/local/share/locale/nl/LC_MESSAGES
/usr/local/share/locale/nl/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/pt_BR
/usr/local/share/locale/pt_BR/LC_MESSAGES
/usr/local/share/locale/pt_BR/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/pt_PT
/usr/local/share/locale/pt_PT/LC_MESSAGES
/usr/local/share/locale/pt_PT/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/sv
/usr/local/share/locale/sv/LC_MESSAGES
/usr/local/share/locale/sv/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/tr
/usr/local/share/locale/tr/LC_MESSAGES
/usr/local/share/locale/tr/LC_MESSAGES/clawsker.mo
/usr/local/share/locale/zh_TW
/usr/local/share/locale/zh_TW/LC_MESSAGES
/usr/local/share/locale/zh_TW/LC_MESSAGES/clawsker.mo
/usr/local/share/man
/usr/local/share/man/man1
/usr/local/share/man/man1/clawsker.1
/usr/share
/usr/share/doc
/usr/share/doc/clawsker-1.3.5
/usr/share/doc/clawsker-1.3.5/AUTHORS.bz2
/usr/share/doc/clawsker-1.3.5/NEWS.bz2
/usr/share/doc/clawsker-1.3.5/README.bz2
```

File: mail-client/clawsker/clawsker-1.3.5.ebuild (hunk 4 above)
Comment (thesamesam) [reply mail-client/clawsker/clawsker-1.3.5.ebuild]: given juippis' comment, I guess the PREFIX part is needed :)

Comment (kuraga) [reply mail-client/clawsker/clawsker-1.3.5.ebuild]: @juppis , @thesamesam , can we set `PREFIX` without `src_install` overload?

Comment (juippis) [reply mail-client/clawsker/clawsker-1.3.5.ebuild]: You can most definitely patch it, but 
```
src_install() {
	emake install DESTDIR="${D}" PREFIX=/usr
}
```
seems simpler in every way. Make sure the docs are being installed, and/or add `einstalldocs` if they're not.

Comment (thesamesam) [reply mail-client/clawsker/clawsker-1.3.5.ebuild]: If upstream's alive, a patch works a bit better because it means we can drop src_install.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**